[2025-04-23T17:42:23.311+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: read_excel_from_mongo_dag_with_rsi.store_in_mysql_task manual__2025-04-23T17:42:19.032169+00:00 [queued]>
[2025-04-23T17:42:23.320+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: read_excel_from_mongo_dag_with_rsi.store_in_mysql_task manual__2025-04-23T17:42:19.032169+00:00 [queued]>
[2025-04-23T17:42:23.321+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-23T17:42:23.334+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): store_in_mysql_task> on 2025-04-23 17:42:19.032169+00:00
[2025-04-23T17:42:23.342+0000] {standard_task_runner.py:60} INFO - Started process 428 to run task
[2025-04-23T17:42:23.348+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'read_excel_from_mongo_dag_with_rsi', 'store_in_mysql_task', 'manual__2025-04-23T17:42:19.032169+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/indicator_dag.py', '--cfg-path', '/tmp/tmpas3hli94']
[2025-04-23T17:42:23.354+0000] {standard_task_runner.py:88} INFO - Job 80: Subtask store_in_mysql_task
[2025-04-23T17:42:23.457+0000] {task_command.py:423} INFO - Running <TaskInstance: read_excel_from_mongo_dag_with_rsi.store_in_mysql_task manual__2025-04-23T17:42:19.032169+00:00 [running]> on host 8291240c5d79
[2025-04-23T17:42:23.757+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='read_excel_from_mongo_dag_with_rsi' AIRFLOW_CTX_TASK_ID='store_in_mysql_task' AIRFLOW_CTX_EXECUTION_DATE='2025-04-23T17:42:19.032169+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-23T17:42:19.032169+00:00'
[2025-04-23T17:42:23.835+0000] {logging_mixin.py:188} INFO - ✅ Database `intermediate` ensured to exist.
[2025-04-23T17:42:23.929+0000] {logging_mixin.py:188} INFO - ❌ MySQL Insert Error: nan can not be used with MySQL
[2025-04-23T17:42:23.930+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-23T17:42:23.966+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=read_excel_from_mongo_dag_with_rsi, task_id=store_in_mysql_task, execution_date=20250423T174219, start_date=20250423T174223, end_date=20250423T174223
[2025-04-23T17:42:24.006+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-23T17:42:24.023+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
